openmls/src/ciphersuite/mod.rs

```diff
@@ -42,8 +43,36 @@ mod tests;
 /// A simple type for HPKE public keys using [`VLBytes`] for (de)serializing.
 pub type HpkePublicKey = VLBytes;
 
+// TODO: derive zeroize
```

Comment:
Is this something you still want to do here? If not, could you create an issue and link it here?

Reply:
Right, will remove and link it in #1240